src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Sorts and lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+    
+    public static final String COMMAND_WORD = "sort";
+    
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts and displays all persons in the address book as a list with index numbers.\n"
```

Comment:
It might be useful to tell the user how the list will be sorted (i.e. alphabetically by first name).